src/actions/action.js

```diff
@@ -34,7 +34,6 @@ export default function action(type, host, path, params = {}, transform) {
       .then(transform || (json => json))
       .then(json => dispatch(getDataOk(json)))
       .catch((e) => {
-        console.error(e);
```

Comment:
I think we want to log the error in this case.

Reply:
Leaving it in would violate the `no-console` rule (https://eslint.org/docs/rules/no-console). Re-adding it and adding a comment above to ignore it for now.